### ISSUE TITLE
Identify point of contact for other region of a potential R submission using pilot1 project

### ISSUE THREAD

**Issue by elong0527** [RConsortium/submissions-wg#65]
It would be great if we could identify point of contact from other region to prepare a potential R submission using `pilot1` work. For example: 

- China
- EU
- Japan
- etc.

**Comment by NNaikp** [comment]
EMA recently published a [data standardisation strategy](https://www.ema.europa.eu/en/documents/other/european-medicines-regulatory-network-data-standardisation-strategy_en.pdf). They are not currently requiring the submission of data in new drug applications but according to this they are investigating and piloting receiving data created in accordance with CDISC.

I will reach out to one of my colleagues in Japan for PMDA.

**Comment by NNaikp** [comment]
For PMDA it is the Office of Advanced Evaluation with Electronic Data that handles everything in regards to eData packages.
Under contacts they only have a phone number but on [this page](https://www.pmda.go.jp/english/review-services/reviews/0002.html) with information on all of the guidelines and forms there is an email address:  jisedaiPT●pmda.go.jp. This is also the email address mentioned in [this update](https://www.pmda.go.jp/files/000215352.pdf) from Yuki Ando in 2016.

My colleague in Japan says that English is generally accepted for the documents in the electronic data package, so a submission would not require a translation of the ADRG. But the cover letter in m1 must probably be in Japanese.

**Comment by lengning** [comment]
thanks @NNaikp ! sounds like a PDMA pilot submission is pretty tangible? do you think we can start a subteam looking into it (I'm happy to reach out to our Chugai (Roche) colleagues as well)?

**Comment by NNaikp** [comment]
Yes, I definitely think we can start a sub-team working on this 👍

It would be nice to discuss this with an RA person with insights into submissions to PMDA. Please reach out to your colleagues - I will also reach out to my RA colleagues in Japan.

As I see it we would pretty much be able to submit the same package as we did for FDA. The only addition would be the creation of an Attachment 8 (former Appendix 8) where they need info on trials, standards, validation software and analysis software. I believe this is translated into Japanese as you usually need to submit a preliminary version of it for any meetings with PMDA.

Can you start up a repo for a package to PMDA? 🙂

**Comment by NNaikp** [comment]
Hi @lengning 

I had a meeting with our RA colleagues and here are some points if we would like to submit the pilot to PMDA:

Access to portal site (submission gateway):
- To access the portal site you must retrieve a certificate from the Medical Information System Development Center. However this certificate is **only allowed to be issued to residents of Japan.** 

Documents:
- The documents to accompany the submission differs depending on what version of the eCTD structure we would like to submit in. Currently PMDA accepts eCTD v3.2.2. and will start support for v4 from April.
- As this package is for testing purposes they didn't think it was necessary to create an Attachment 8 as this is not part of submitting a data package to PMDA.
- A cover letter in Japanese must be included in m1.
- It is acceptable that documents in the eData package are in English.

eData package:
- If the submission is done according to eCTD v3.2.2 then the data packages are not part of eCTD but "attached" as a supplemental in the portal site and a metadata file (TSV file) must be provided in m4.
- If we are allowed to submit in eCTD v4 then the data package would go into m5.

----

**In short:**
The package used for FDA can almost in its entirety be used for a PMDA submission and would only require:
- [ ] Initiation of discussion with Office for Advanced Evaluation with Electronic Data (jisedaiPT@pmda.go.jp)
- [ ] Modification and translation of cover letter to Japanese
- [ ] Help from a Japanese colleague/member to acquire access to portal site
- [ ] Creation of TSV file depending of version of eCTD we can agree to with PMDA

**Comment by hongyanregngithub** [comment]
PMDA accepts R codes. In addition to the data submitted to the FDA, output files and files on simulation should also be submitted to PMDA. Please refer to section 4.2.3 of Technical Conformance Guide on Electronic Study Data Submissions (dated Jan 24, 2019) described
Other relevant notices are listed below. You can refer to them from the link, but please let me know if you cannot access it. 
関連通知・様式等 | 独立行政法人 医薬品医療機器総合機構 (pmda.go.jp)
	Basic Principles on Electronic Data Submission of Study Data for J-NDA (Notification No. 620-6, dated June 20, 2014)
	Question and Answer Guide (Q&A) Regarding [Basic Principles for Electronic Data Submission of Study Data for J-NDA] (dated March 18, 2020)
	Practical Operations of Electronic Study Data Submissions (Notification No. 427-1 dated April 27, 2015)
	Question and Answer Guide (Q&A) Regarding [Practical Operations of Electronic Study Data Submissions] (dated March 18, 2020)
I am still waiting for the answer on if a direct contact of PMDA reviewer and will share if it is available.
